tests/end-to-end/backup/pkg/common/common.go

```diff
@@ -72,9 +71,6 @@ func RunTest(t *testing.T, mode TestMode) {
 	helmBrokerTest, err := servicecatalog.NewHelmBrokerTest()
 	fatalOnError(t, err, "while creating structure for HelmBroker test")
 
-	myEventBusTest, err := eventbus.NewEventBusTest()
-	fatalOnError(t, err, "while creating structure for EventBus test")
-
```

Comment:
https://github.com/kyma-project/kyma/pull/7756/files#diff-7c8754c94a647df85f1b987aa53d14ddL84
This does not test the compatibility layer which is used by most of the clients.

Reply:
i cannot see what you want to show me with that link. But i agree: this does not test the compatibility layer anymore.
Lets update the eventmesh test then so that it covers both scenarios.

Reply:
done